### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by deyantara1303** [Navidfoumani/ConvTran#15]
Do the models assume that the classes are balanced?
Will they work for imbalanced dataset?

**Comment by Navidfoumani** [comment]
We didn’t specifically design the models for highly imbalanced datasets since the benchmarks we tested were pretty balanced. 

But they should still work—you might just need to tweak the loss function. You can try replacing cross-entropy loss with something like **Focal Loss** or **Weighted Cross-Entropy** to handle the imbalance better. Let us know if you need any help with that!
